ruff.toml

```diff
@@ -36,9 +35,14 @@ ignore = [
     "E721",
     "E731",
     "E741",
+    "F403",  # `from ... import *` used; unable to detect undefined names
+    "F405",  # may be undefined, or defined from star imports
```

Comment:
In my experience these errors usually occur in `__init__.py` files. Perhaps it could be moved to the `per-file-ignores`?

Reply:
Let me give it a try.

Reply:
I'm still getting 487 errors instead of 562, only 75 of them originate in `__init__.py` files. However, they occur in only 8 files, for which I can easily add per-file excludes:
```
numpy/_core/defchararray.py
numpy/_core/multiarray.py
numpy/_core/numeric.py
numpy/_core/umath.py
numpy/f2py/capi_maps.py
numpy/f2py/crackfortran.py
numpy/f2py/f90mod_rules.py
numpy/matlib.py
```